src/StreakerGame.java

```diff
@@ -45,22 +44,21 @@ public static void main(String[] args) {
 
     @Override
     public void start(Stage stage) {
-        movingSpeed = new IntValue(constants.getStartingSpeed());
         setupGameState(stage);
         new AnimationTimer() {
             public void handle(long currentNanoTime) {
-                double elapsedTime = (currentNanoTime - lastNanoTime.value) / constants.getPrecision();
+                double elapsedTime = (currentNanoTime - lastNanoTime.value) / Constants._PRECISION;
                 lastNanoTime.value = currentNanoTime;
                 double nanot = currentNanoTime - startNanoTime;
-                double t = nanot / constants.getPrecision();
+                double t = nanot / Constants._PRECISION;
                 background.loop();
                 background.setBackgroundSpeed(gc);
                 character.handleVelocity(input);
                 character.handleCharacterPosition();
                 character.updateVelocity(elapsedTime);
                 character.render(gc, t);
                 for (Coin coin : coins) {
-                    if (coin.getY() > constants.getScreenHeight()) {
```

Comment:
Lines 60-69 got moved into a method called handleCoinIntersects. Should not change anything you did.